src/modinfoforeign.cpp

```diff
@@ -49,7 +42,15 @@ ModInfoForeign::ModInfoForeign(const QString& modName, const QString& referenceF
     : ModInfoWithConflictInfo(core), m_ReferenceFile(referenceFile),
       m_Archives(archives), m_ModType(modType)
 {
-  m_CreationTime = QFileInfo(referenceFile).birthTime();
+  m_CreationTime          = QFileInfo(referenceFile).birthTime();
+  IPluginGame const* game = qApp->property("managed_game").value<IPluginGame*>();
+  QMap<QString, QDir> directories = {{"data", game->dataDirectory()}};
+  directories.insert(game->secondaryDataDirectories());
+  for (QDir directory : directories) {
+    if (referenceFile.startsWith(directory.absolutePath(), Qt::CaseInsensitive)) {
```

Comment:
Maybe there is a better way in Qt to check if a file is under a directory?

Reply:
A quick search didn't find anything but there may be a way.

Reply:
Still haven't found a better solution for this. It seems to work.